source/SpinalHDL/Data types/bits.rst

```diff
@@ -330,3 +330,18 @@ Misc
    myBits_8bits  := myBits_32bits.resized       // automatic resize (myBits_8bits = 0x44)
    myBits_8bits  := myBits_32bits.resize(8)     // resize to 8 bits (myBits_8bits = 0x44)
    myBits_8bits  := myBits_32bits.resizeLeft(8) // resize to 8 bits (myBits_8bits = 0x11)
+
+.. _maskedlitteral:
+
+MaskedLitteral
+~~~~~~~~~~~~~~
+
+MaskedLitteral values are bit vectors with don’t care values denoted with `-`.
+
+.. code-block:: scala
+
+     val myBits = B"1101"
+
+     val test1 = RegInit(myBits === M"1-01") // True
```

Comment:
Hmm maybe RegNext instead of RegInit ?
What about just having no RegXXX at all to keep the example "raw" ?

Reply:
Yeah you are obviously right, I don’t know what I was thinking